### ISSUE TITLE
yarn install fails to execute on docker script

### ISSUE THREAD

**Issue by diegogd** [leemonade/leemons#5]
## Initial conditions

It is first execution, so `../examples/docker/monorepo_node_modules` is empty.

## Context

There is a race condition in the initialization of the docker-compose environment.

During the first execution of `docker/start.sh`:
```bash
docker compose --env-file ../examples/docker/.env up -d --remove-orphans
```
The `leemons-yarn-install` container die and following line doesn't executes:
```bash
docker compose exec leemons-yarn-install sh installDeps.sh
```

## Solution

One solution is running first this task with a new container
```
docker compose run leemons-yarn-install sh installDeps.sh
```

That let the dependencies to be initialized and the container `leemons-yarn-install` can run executing `./start.sh` on `docker` folder.

**Comment by diegogd** [comment]
Can be another problem with `../examples/docker/monorepo_yarn.lock` as when this volume is linked to the docker, it is initialized as a folder, so it would be required to create first this file.

For example with this initialization:

```bash
if [ ! -f ../examples/docker/monorepo_yarn.lock ]
then
    touch ../examples/docker/monorepo_yarn.lock
    docker compose run leemons-yarn-install sh installDeps.sh
fi
docker compose --env-file ../examples/docker/.env up -d --remove-orphans
docker compose exec leemons-yarn-install sh installDeps.sh
docker compose logs -f
```

**Comment by MIGUELez11** [comment]
I'm going to investigate about it, the proposed solutions looks nice to me.

The leemons-yarn-install command execution is to display the yarn installation progress

**Comment by diegogd** [comment]
I dont know if this can be related, but the `/monorepo/yarn.lock` mountpoint is duplicated at `docker-compose.yaml` file. In new versions of docker command with compose integrated it didn't raise any exception but in docker-compose previous versions it did.

https://github.com/leemonade/leemons/blob/5c3c1ba31d15d084ac1fea527569d93baaec9f3c/docker/docker-compose.yaml#L10-L11

If the first line is removed the proposed solution can be discarded.

**Comment by MIGUELez11** [comment]
I'm removing the duplicated line.

The good one is the first one, since the yarn installation on linux might change it and will result in continuously installing dependencies across environment changes.

**Comment by MIGUELez11** [comment]
The proposed solution solves the `EISDIR` error when installing yarn dependencies. Still, I'm working on having a better docker solution for now :D.

Thank you so much for your contributions.

**Comment by MIGUELez11** [comment]
Fixed on accc233, also added the solution to the new powershell script.
